### ISSUE TITLE
安装gpu又惨败，提示什么页面太大不得不强行关闭

### ISSUE THREAD

**Issue by b95595** [babysor/MockingBird#673]
显卡Rtx 2060  cuda_11.6

各种尝试都不行 搞了三天了  开始用cpu跑 觉得太慢 想用显卡跑结果各种安装失败错误  关键是这显卡不上不下 11.6  也不知道哪里出错了  哪位好心人给个详细教程吧   拜托了 真的很喜欢这个  6679314@qq.com 邮箱

**Comment by pzhyyd** [comment]
訓練模型顯存不足
訓練合成器時：將 synthesizer/hparams.py中的batch_size參數調小

//調整前
tts_schedule = [(2, 1e-3, 20_000, 12), # Progressive training schedule
(2, 5e-4, 40_000, 12), # (r, lr, step, batch_size)
(2, 2e-4, 80_000, 12), #
(2, 1e-4, 160_000, 12), # r = reduction factor (# of mel frames
(2, 3e-5, 320_000, 12), # synthesized for each decoder iteration)
(2, 1e-5, 640_000, 12)], # lr = learning rate
//調整後
tts_schedule = [(2, 1e-3, 20_000, 8), # Progressive training schedule
(2, 5e-4, 40_000, 8), # (r, lr, step, batch_size)
(2, 2e-4, 80_000, 8), #
(2, 1e-4, 160_000, 8), # r = reduction factor (# of mel frames
(2, 3e-5, 320_000, 8), # synthesized for each decoder iteration)
(2, 1e-5, 640_000, 8)], # lr = learning rate

**Comment by b95595** [comment]
搞好了&nbsp; 谢谢&nbsp; 不过我是在cmd里面调的&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "babysor/MockingBird"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月24日(星期天) 晚上7:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [babysor/MockingBird] 安装gpu又惨败，提示什么页面太大不得不强行关闭 (Issue #673)





 
訓練模型顯存不足
 訓練合成器時：將 synthesizer/hparams.py中的batch_size參數調小
 
//調整前
 tts_schedule = [(2, 1e-3, 20_000, 12), # Progressive training schedule
 (2, 5e-4, 40_000, 12), # (r, lr, step, batch_size)
 (2, 2e-4, 80_000, 12), #
 (2, 1e-4, 160_000, 12), # r = reduction factor (# of mel frames
 (2, 3e-5, 320_000, 12), # synthesized for each decoder iteration)
 (2, 1e-5, 640_000, 12)], # lr = learning rate
 //調整後
 tts_schedule = [(2, 1e-3, 20_000, 8), # Progressive training schedule
 (2, 5e-4, 40_000, 8), # (r, lr, step, batch_size)
 (2, 2e-4, 80_000, 8), #
 (2, 1e-4, 160_000, 8), # r = reduction factor (# of mel frames
 (2, 3e-5, 320_000, 8), # synthesized for each decoder iteration)
 (2, 1e-5, 640_000, 8)], # lr = learning rate
